use_cases/gretel.json

```diff
@@ -2,11 +2,12 @@
   "cards": [
     {
       "gtmId": "use-case-synthetic",
-      "title": "Generate synthetic data",
-      "description": "Create highly accurate synthetic training data for ML models.",
+      "title": "Generate synthetic tabular data",
```

Comment:
should this usecase get a 'new' tag as well?

Reply:
it's not a new card. I just updated the text and added a details page.